インストーラー作成時に大量の警告が出る

PS D:\OneDrive\交通系\src\ICCardManager\installer> powershell -ExecutionPolicy Bypass -File .\build-installer.ps1
======================================
 ICCardManager インストーラービルド
======================================

[1/6] Inno Setup の確認...
  Inno Setup: C:\Users\masay\AppData\Local\Programs\Inno Setup 6\ISCC.exe

[2/6] アイコンの生成...
======================================
 アプリケーションアイコン生成
======================================

[1/2] アイコンを生成中...
[2/2] ICOファイルに変換中...

======================================
 生成完了!
======================================

出力ファイル: D:\OneDrive\交通系\src\ICCardManager\installer\..\src\ICCardManager\Resources\app.ico
インストーラー用: D:\OneDrive\交通系\src\ICCardManager\installer\app.ico

  アイコン生成完了: D:\OneDrive\交通系\src\ICCardManager\installer\app.ico

[3/6] アプリケーションのビルド...
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Data\DbContext.cs(94,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [D:\OneDr
ive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Data\DbContext.cs(239,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [D:\OneD
rive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Views\MainWindow.xaml.cs(57,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [D
:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Views\MainWindow.xaml.cs(100,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Views\MainWindow.xaml.cs(156,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Data\Migrations\MigrationRunner.cs(347,30): warning CS0168: 変数 'ex' は宣言されていますが、使
用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\Logging\FileLoggerProvider.cs(113,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\Logging\FileLoggerProvider.cs(193,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\Logging\FileLoggerProvider.cs(229,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(733,30): warning CS0168: 変数 'ex' は宣言
されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(847,30): warning CS0168: 変数 'ex' は宣言
されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(938,30): warning CS0168: 変数 'ex' は宣言
されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(1056,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ViewModels\BusStopInputViewModel.cs(164,26): warning CS0168: 変数 'ex' は宣言されていますが、
使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Services\StationMasterService.cs(101,34): warning CS0168: 変数 'ex' は宣言されていますが、使用さ
れていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ViewModels\PrintPreviewViewModel.cs(230,30): warning CS0168: 変数 'ex' は宣言されていますが、
使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager_prvnjohz_wpftmp.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Data\DbContext.cs(94,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [D:\OneDr
ive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Data\DbContext.cs(239,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [D:\OneD
rive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Views\MainWindow.xaml.cs(57,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [D
:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Views\MainWindow.xaml.cs(100,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Views\MainWindow.xaml.cs(156,30): warning CS0168: 変数 'ex' は宣言されていますが、使用されていません [
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Data\Migrations\MigrationRunner.cs(347,30): warning CS0168: 変数 'ex' は宣言されていますが、使
用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\Logging\FileLoggerProvider.cs(113,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\Logging\FileLoggerProvider.cs(193,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\Logging\FileLoggerProvider.cs(229,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ViewModels\BusStopInputViewModel.cs(164,26): warning CS0168: 変数 'ex' は宣言されていますが、
使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Services\StationMasterService.cs(101,34): warning CS0168: 変数 'ex' は宣言されていますが、使用さ
れていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(733,30): warning CS0168: 変数 'ex' は宣言
されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(847,30): warning CS0168: 変数 'ex' は宣言
されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(938,30): warning CS0168: 変数 'ex' は宣言
されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\Infrastructure\CardReader\PcScCardReader.cs(1056,30): warning CS0168: 変数 'ex' は宣
言されていますが、使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ViewModels\PrintPreviewViewModel.cs(230,30): warning CS0168: 変数 'ex' は宣言されていますが、
使用されていません [D:\OneDrive\交通系\src\ICCardManager\src\ICCardManager\ICCardManager.csproj]
  ビルド完了: D:\OneDrive\交通系\src\ICCardManager\publish

[4/6] リソースファイルのコピー...

--- a/ICCardManager/src/ICCardManager/Data/DbContext.cs
+++ b/ICCardManager/src/ICCardManager/Data/DbContext.cs
@@ -93,6 +93,7 @@ namespace ICCardManager.Data
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
                 // 権限設定に失敗してもディレクトリ作成は試みる
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[DbContext] ディレクトリ権限設定エラー: {ex.Message}");
@@ -238,6 +239,7 @@ namespace ICCardManager.Data
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
                 // アクセス権限の設定に失敗してもアプリケーションは続行
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[DbContext] アクセス権限の設定に失敗: {ex.Message}");

--- a/ICCardManager/src/ICCardManager/Data/Migrations/MigrationRunner.cs
+++ b/ICCardManager/src/ICCardManager/Data/Migrations/MigrationRunner.cs
@@ -346,6 +346,7 @@ VALUES (@operator_idm, @operator_name, @target_table, @target_id, @action, @afte
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
                 // ログ記録の失敗はマイグレーション自体には影響させない
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[Migration] Failed to log migration action: {ex.Message}");

--- a/ICCardManager/src/ICCardManager/Infrastructure/CardReader/PcScCardReader.cs
+++ b/ICCardManager/src/ICCardManager/Infrastructure/CardReader/PcScCardReader.cs
@@ -732,6 +732,7 @@ namespace ICCardManager.Infrastructure.CardReader
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"再接続試行エラー: {ex.Message}");
 #endif
@@ -846,6 +847,7 @@ namespace ICCardManager.Infrastructure.CardReader
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"Pollingコマンドエラー: {ex.Message}");
 #endif
@@ -937,6 +939,7 @@ namespace ICCardManager.Infrastructure.CardReader
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"ブロック{blockIndex}読み取り失敗: {ex.Message}");
 #endif
@@ -1055,6 +1058,7 @@ namespace ICCardManager.Infrastructure.CardReader
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"履歴データのパースエラー: {ex.Message}");
 #endif

--- a/ICCardManager/src/ICCardManager/Infrastructure/Logging/FileLoggerProvider.cs
+++ b/ICCardManager/src/ICCardManager/Infrastructure/Logging/FileLoggerProvider.cs
@@ -112,6 +112,7 @@ namespace ICCardManager.Infrastructure.Logging
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[FileLogger] Failed to write log: {ex.Message}");
 #endif
@@ -192,6 +193,7 @@ namespace ICCardManager.Infrastructure.Logging
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[FileLogger] Failed to cleanup old logs: {ex.Message}");
 #endif
@@ -228,6 +230,7 @@ namespace ICCardManager.Infrastructure.Logging
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
                 // 権限設定に失敗してもディレクトリ作成は試みる
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[FileLogger] ディレクトリ権限設定エラー: {ex.Message}");

--- a/ICCardManager/src/ICCardManager/Services/StationMasterService.cs
+++ b/ICCardManager/src/ICCardManager/Services/StationMasterService.cs
@@ -100,6 +100,7 @@ namespace ICCardManager.Services
                 }
                 catch (Exception ex)
                 {
+                    _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                     System.Diagnostics.Debug.WriteLine($"駅マスタ読み込みエラー: {ex.Message}");
 #endif

--- a/ICCardManager/src/ICCardManager/ViewModels/BusStopInputViewModel.cs
+++ b/ICCardManager/src/ICCardManager/ViewModels/BusStopInputViewModel.cs
@@ -163,6 +163,7 @@ public partial class BusStopInputViewModel : ViewModelBase
         }
         catch (Exception ex)
         {
+            _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
             System.Diagnostics.Debug.WriteLine($"[BusStopInput] サジェスト候補の読み込みに失敗: {ex.Message}");
 #endif

--- a/ICCardManager/src/ICCardManager/ViewModels/PrintPreviewViewModel.cs
+++ b/ICCardManager/src/ICCardManager/ViewModels/PrintPreviewViewModel.cs
@@ -229,6 +229,7 @@ public partial class PrintPreviewViewModel : ViewModelBase
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
                 // ドキュメントがまだビジュアルツリーにアタッチされていない場合など
                 // ページ数の取得に失敗することがある
 #if DEBUG

--- a/ICCardManager/src/ICCardManager/Views/MainWindow.xaml.cs
+++ b/ICCardManager/src/ICCardManager/Views/MainWindow.xaml.cs
@@ -56,6 +56,7 @@ namespace ICCardManager.Views
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
                 // 終了時のエラーは警告のみ（アプリ終了を妨げない）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[MainWindow] 終了時エラー: {ex.Message}");
@@ -99,6 +100,7 @@ namespace ICCardManager.Views
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[MainWindow] ウィンドウ位置の保存に失敗: {ex.Message}");
 #endif
@@ -155,6 +157,7 @@ namespace ICCardManager.Views
             }
             catch (Exception ex)
             {
+                _ = ex; // 警告抑制（DEBUGビルドでのみ使用）
 #if DEBUG
                 System.Diagnostics.Debug.WriteLine($"[MainWindow] ウィンドウ位置の復元に失敗: {ex.Message}");
 #endif

